include/z64message.h

```diff
@@ -23,6 +23,37 @@ typedef enum {
 // TODO get these properties from the textures themselves
 #define MESSAGE_TEXTURE_STATIC_TEX_SIZE 0x900
 
+// Macros for generating characters in the filename encoding (specified by message 0xFFFC
+// and loaded by Font_LoadOrderedFont). For example, FILENAME_UPPERCASE('A') will encode
+// the character 'A'.
+#if OOT_NTSC
+#define FILENAME_DIGIT(c)                   ((c) - '0')
+// 0x0A - 0x59: hirigana
+// 0x5A - 0xAA: katakana
+#define FILENAME_UPPERCASE(c)               ((c) - 'A' + 0xAB)
+#define FILENAME_LOWERCASE(c)               ((c) - 'a' + 0xC5)
+#define FILENAME_SPACE                      0xDF
+// 0xE0: unknown
+#define FILENAME_QUESTION_MARK              0xE1
+#define FILENAME_EXCLAMATION_MARK           0xE2
+#define FILENAME_COLON                      0xE3
+#define FILENAME_DASH                       0xE4
+#define FILENAME_LEFT_PARENTHESES           0xE5
+#define FILENAME_RIGHT_PARENTHESES          0xE6
```

Comment:
```suggestion
#define FILENAME_LEFT_PARENTHESIS           0xE5
#define FILENAME_RIGHT_PARENTHESIS          0xE6
```

https://en.wikipedia.org/wiki/Bracket#Parentheses_or_round_brackets

Reply:
yeah I agree, it's called "parentheses" in the kanji file and nes_font_static currently though and I want to stay consistent with that

Reply:
Oh ok sure 👍